Cartfile

```diff
@@ -1,2 +1,2 @@
-github "DeclarativeHub/ReactiveKit" ~> 3.8
+github "DeclarativeHub/ReactiveKit" ~> 3.9
 github "tonyarnold/Differ" ~> 1.0
```

Comment:
You'll want to move Differ up to `~> 1.1.0` for the Xcode 9.3/Swift 4.1 compatibility.